arangod/Aql/QueryRegistry.h

```diff
@@ -142,6 +146,9 @@ class QueryRegistry {
 
     std::shared_ptr<ClusterQuery> _query;  // the actual query pointer
 
+    bool _finished = false;
```

Comment:
Can we move this new Boolean member further down, directly in front of or after the other Boolean `_isTombstone` member?
This would make the struct not grow in memory usage, whereas placing the Boolean here will make it grow by 8 bytes.